package.json

```diff
@@ -16,7 +16,7 @@
     "build:ios:ext:prebuilt": "webextensiom-shim-prebuilt",
     "build:ios:webpack": "build iOS",
     "lint": "eslint -c packages/.eslintrc.json packages --ext .ts,.tsx --fix --cache",
-    "lint:report": "eslint -c packages/.eslintrc.json packages --ext .ts,.tsx --format junit -o reports/junit/eslint-results.xml --cache",
```

Comment:
`--format junit -o reports/junit/eslint-results.xml` this part is code coveraging

Reply:
The line move to ci